Drop unused ze_pfn*ExpCb_t typedefs that clash with level-zero >=1.33
(ze_loader.h now pulls in layers/zel_tracing_register_cb.h, which
defines the same names with a different signature).

--- a/level_zero/include/level_zero/driver_experimental/mcl_ext/ze_mutable_command_list_exp.h	2026-10-08 06:47:46.000000000 -0700
+++ b/level_zero/include/level_zero/driver_experimental/mcl_ext/ze_mutable_command_list_exp.h	2026-10-16 23:46:59.165720818 -0700
@@ -8,36 +8,3 @@
 #pragma once
 
 #include <level_zero/ze_api.h>
-
-typedef ze_result_t(ZE_APICALL *ze_pfnCommandListGetNextCommandIdExpCb_t)(
-    ze_command_list_handle_t hCommandList,
-    const ze_mutable_command_id_exp_desc_t *desc,
-    uint64_t *pCommandId);
-
-typedef ze_result_t(ZE_APICALL *ze_pfnCommandListUpdateMutableCommandsExpCb_t)(
-    ze_command_list_handle_t hCommandList,
-    const ze_mutable_commands_exp_desc_t *desc);
-
-typedef ze_result_t(ZE_APICALL *ze_pfnCommandListUpdateMutableCommandSignalEventExpCb_t)(
-    ze_command_list_handle_t hCommandList,
-    uint64_t commandId,
-    ze_event_handle_t hSignalEvent);
-
-typedef ze_result_t(ZE_APICALL *ze_pfnCommandListUpdateMutableCommandWaitEventsExpCb_t)(
-    ze_command_list_handle_t hCommandList,
-    uint64_t commandId,
-    uint32_t numWaitEvents,
-    ze_event_handle_t *phWaitEvents);
-
-typedef ze_result_t(ZE_APICALL *ze_pfnCommandListGetNextCommandIdWithKernelsExpCb_t)(
-    ze_command_list_handle_t hCommandList,
-    const ze_mutable_command_id_exp_desc_t *desc,
-    uint32_t numKernels,
-    ze_kernel_handle_t *phKernels,
-    uint64_t *pCommandId);
-
-typedef ze_result_t(ZE_APICALL *ze_pfnCommandListUpdateMutableCommandKernelsExpCb_t)(
-    ze_command_list_handle_t hCommandList,
-    uint32_t numKernels,
-    uint64_t *pCommandId,
-    ze_kernel_handle_t *phKernels);
